subsys/bluetooth/controller/ll_sw/lll_sync.h

```diff
@@ -10,6 +10,10 @@ struct lll_sync {
 	uint8_t access_addr[4];
 	uint8_t crc_init[3];
 
+	uint8_t phy:3;
+	uint8_t is_rx_enabled:1;
+	uint8_t is_aux_sched:1;
```

Comment:
`is_aux_sched` means, `lll_sync` has used LLL scheduling to receive a auxiliary channel PDU (for Periodic Adv it is AUX_CHAIN_IND). The naming is consistent with `is_aux_sched` in lll_scan context.